server/routes/employee.js

```diff
@@ -3,40 +3,69 @@ const router = express.Router();
 const Employee = require('../models/Employee');
 
 /*
-    TODO check if retunr object needs to be formatted
-    Gets an employee from a email
+    * DONE
+    ? Does the implementation of employee need to change when we start logging them into tills? Should we should their objectId?
+    Gets an employee from an email
 */
-router.get('/get', function(req, res){
+router.post('/get', function(req, res){
+    //Check if req body exists
+    if(!req.body) return res.status(400).send({err: 'No request body', code: 400});
+
     let email = req.body.email;
+
+    //Attempt to find employee and return it
     Employee.findOne({email: email}, function(err, employee){
         if(err){
             console.log(err);
+            return res.status(500).send({err: 'Unable to find employee', code: 500});
         } else {
-            res.json(employee);
+            //If employee not found
+            if(employee === null) return res.status(404).send({err: `Employee does not exist`, code: 404});
+            let formattedEmployee = {
+                id: employee._id,
+                email: employee.email,
+                isManager: employee.isManager,
+                employeeId: employee.employeeId
+            };
+            return res.status(201).send({formattedEmployee, code: 201});
         }
     })
 });
 
 /*
-    Posts an employee with the email and isManager
+    * DONE
+    Posts an employee with an email and isManager
 */
 router.post('/create', async (req, res) => {
+    //check if req body exists 
     if(!req.body) return res.status(400).send({err: 'No request body'});
```

Comment:
Just a general comment, I think we do this everywhere. This only checks that there is indeed a body. If they give a bad body in the request some error will fire that we may not be checking for or worse we might make an incorrect db entry.